tests/app/questionnaire/rules/test_rule_evaluator.py

```diff
@@ -854,7 +854,9 @@ def test_answer_source_default_answer_used_when_no_answer(
 def test_raises_exception_when_bad_operand_type():
     with pytest.raises(TypeError):
         rule_evaluator = get_rule_evaluator()
-        rule_evaluator.evaluate(rule={Operator.EQUAL: {1, 1}})
+        rule_evaluator.evaluate(
+            rule={Operator.EQUAL: {1, 1}}  # pylint: disable=duplicate-value
```

Comment:
The issue should be fixed.

Reply:
Updated, please check if the change I made was what is expected with this test.

Reply:
Updated to `rule={Operator.EQUAL: {1, 2}}`. Thanks for the suggestion 👍